modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/StandaloneCliOptions.cs

```diff
@@ -44,6 +44,11 @@ public class StandaloneCliOptions : Dictionary<string, string>, IAgentConfigProv
         /// <param name="args">The specified command line arguments.</param>
         public StandaloneCliOptions(string[] args) {
 
+            bool showHelp = false;
+            bool isLegacyOption = false;
+            List<string> legacyOptions = new List<string>();
+            var logger = ConsoleLogger.Create(LogEventLevel.Warning);
```

Comment:
Can we make sure that we have same log format as in the case of usual logs, such as having timestamp and information on source ?

Reply:
same output
![image](https://user-images.githubusercontent.com/7942439/158570264-66c14ae8-8830-415a-9d0a-c456ef44d780.png)